test/basic-querying.test.js

```diff
@@ -201,9 +206,11 @@ describe('basic-querying', function() {
       User.find({order: 'order'}, function(err, users) {
```

Comment:
For Cassandra, the sorting field (`order` in this case) needs to be defined as part of the clustering key. How do we express that metadata in the model definition?